test/dotnet-grpc.Tests/TestBase.cs

```diff
@@ -16,21 +16,27 @@
 
 #endregion
 
-using System;
+using System.Collections.Generic;
 using System.Net;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using Grpc.Dotnet.Cli.Commands;
 using Microsoft.Build.Locator;
 using NUnit.Framework;
 
 namespace Grpc.Dotnet.Cli.Tests
 {
     public class TestBase
     {
+
         internal static readonly string SourceUrl = "https://contoso.com/greet.proto";
 
-        internal static readonly string ProtoContent = @"// Copyright 2019 The gRPC Authors
+        internal static Dictionary<string, string> ContentDictionary = new Dictionary<string, string>()
```

Comment:
I'm not a fan of a mutable static dictionary that is shared between tests. Keep tests isolated.

Create and configure TestMessageHandler setup as needed inside unit tests that use it.